### PR TITLE
Feature request Github Issue Form

### PR DESCRIPTION
**Pull request by smk4664**
Github has a new beta feature for Issue Forms, I thought it would be nice
to convert our Feature Request template to this new form.  You can see the
new form in action at https://github.com/smk4664/nautobot/blob/feature_request_form/.github/ISSUE_TEMPLATE/feature_request_beta.yaml

### CODE REVIEW

**Comment by bryanculver** [comment]
@smk4664: Is there a way to get the examples into the boxes? Not pre-filled but as example test.

image

The value of the current format is it provides an example to users as to what we would like to see in a feature request submission:

image

Also, please remove the `___` in the input headings (`WHAT, ...` instead of `___WHAT, ...___`). Since plain text doesn't support styling the underscores are used to highlight the user's input. This becomes unnecessary with the GitHub Form as it supports styling as well as selections.

**Comment by smk4664** [comment]
I think the examples would then be `pre-filled` and allow submission without completing, but I will try it out. If not, I could put the examples in Markdown.

**Comment by bryanculver** [comment]
This is referred to as "placeholder" in the `<input>` attribute:


iu-2

**Comment by smk4664** [comment]
I was about to leave a comment that I found the `placeholder` option.

File: .github/ISSUE_TEMPLATE/feature_request_beta.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    attributes:
+      value: >
+        **NOTE:** IF YOUR ISSUE DOES NOT FOLLOW THIS TEMPLATE, YOU WILL BE REQUIRED TO
+        UPDATE THE ISSUE IN ORDER TO HAVE IT REVIEWED. This form is only for proposing
+        specific new features or enhancements. If you have a general idea or question,
+        please start a [discussion](https://github.com/nautobot/nautobot/discussions) instead.
+  - type: dropdown
+    id: persona
+    attributes:
+      label: As a WHO, the user persona___,
```
Comment (glennmatthews) [inline .github/ISSUE_TEMPLATE/feature_request_beta.yaml:16]:
```suggestion
      label: As WHO, the USER PERSONA,
```

**Review by glennmatthews — COMMENTED** [review]
Cool stuff!

File: .github/ISSUE_TEMPLATE/feature_request_beta.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: I know this is done when... HOW
+      description: The requirements/acceptance criteria (a list) as to how this functionality should be implemented.
+      placeholder: |
+        - I create a Nautobot plugin and have the option to inject it in the Plugins
+          dropdown menu or create a new dedicated dropdown in the navigation bar.
+        - When multiple plugins provide dedicated dropdowns, I can re-order the plugin
+          dropdowns in the navbar.
```
Comment (bryanculver) [inline .github/ISSUE_TEMPLATE/feature_request_beta.yaml:61]:
```suggestion
        - I create a Nautobot plugin and have the option to inject it in the Plugins
          dropdown menu or create a new dedicated dropdown in the navigation bar.
          
        - When multiple plugins provide dedicated dropdowns, I can re-order the plugin
          dropdowns in the navbar.
```

Still appears to be rendering run together. Does separating this with another newline make them appear as a list in the text area?

image

Comment (smk4664) [reply .github/ISSUE_TEMPLATE/feature_request_beta.yaml]: This is an issue with Safari and Placeholders. https://github.com/github/feedback/discussions/4290

Comment (smk4664) [reply .github/ISSUE_TEMPLATE/feature_request_beta.yaml]: More information about the issue can be found https://github.com/tests-always-included/angular-placeholder/issues/14#issuecomment-202400788

**Review by bryanculver — APPROVED** [review]
LGTM! Thanks @smk4664!

Gonna do a quick test as beta (since we can't test as a fork) and if it works we'll move this to the main template type.

**Pull request merged**